src/commons/BrowserUtils.ts

```diff
@@ -805,95 +741,78 @@ export namespace BrowserUtils {
-   * @param xOffset - X (Pixels) offset to move to, relative to the top-left corner of the element If not specified, the mouse will move to the middle of the element.
-   * @param yOffset - Y (Pixels) offset to move to, relative to the top-left corner of the element. If not specified, the mouse will move to the middle of the element.
+   * Move the mouse by an offset of the specified element.
+   * If no element is specified, the move is relative to the current mouse cursor.
+   * If an element is provided but no offset, the mouse will be moved to the center of the element.
+   * If the element is not visible, it will be scrolled into view
+   * @param selector element to move to, If not specified or is null, the offset is relative to current position of the mouse.
+   * @param options moveTo command options. options example {xOffset: 5, yOffset: 6}
    */
-  export function moveMouseCursorTo(selector: string, xOffset: number, yOffset: number): void {
-    Reporter.debug(`Move mouse cursor to element: '${selector}' with offset '${xOffset},${yOffset}'`);
+  export function moveTo(selector: string, options?: MoveToOptions): void {
+    Reporter.debug(`Move mouse cursor to element: '${selector}' with offset '${JSON.stringify(options)}'`);
 
-    isExist(selector);
-    $(selector).moveTo({
-      xOffset,
-      yOffset,
-    });
+    waitForExist(selector);
+    $(selector).moveTo(options);
   }
 
   /**
-   * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+   * Releases the mouse button previously held (where the mouse is currently at).
+   * Must be called once for every buttondown command issued.
+   * See the note in click and buttondown about implications of out-of-order commands.
+   * @param mouseButton enum: LEFT = 0, MIDDLE = 1, RIGHT = 2, defaults to the left mouse button if not specified
    */
-  export function releaseMouseButton(mouseButton: number): void {
-    // Defaults to the left mouse button if not specified.
-    const selectedMouseButton: number = mouseButton === undefined ? MouseButton.LEFT : mouseButton;
-    Reporter.step(`Release mouse button '${selectedMouseButton}'`);
-    browser.buttonUp(selectedMouseButton);
+  export function buttonUp(mouseButton: number): void {
+    Reporter.step(`Release mouse button '${mouseButton ? MouseButton.LEFT : mouseButton}'`);
+    browser.buttonUp(mouseButton);
   }
 
   /**
    * Determine an element’s location on the page. The point (0pix, 0pix) refers to the upper-left corner of the page.
-   * @param selector  - element with requested position offset
+   * @param selector element with requested position offset
    */
-  export function getElementLocation(selector: string): Location {
-    Reporter.debug(`Get Element location '${selector}'`);
+  export function getLocation(selector: string): Location {
+    Reporter.debug(`Get element's location '${selector}'`);
 
     return $(selector).getLocation();
   }
```

Comment:
accroding to wdio page you should add prop `https://webdriver.io/docs/api/element/getLocation/`

Reply:
there is a mismatch between docs and actual function signature. anyway don't see a reason to get only part of the data